### ISSUE TITLE
no changes on exported video

### ISSUE THREAD

**Issue by yakuv0** [Akascape/FF-Dissolve-Glitch#3]
hello

seems like its an awesome project, though facing issues with usage.

downloading the example file and converting it to mp4 then use the tool with default options resulted with the same exact input file visually.
another example with the same results is a concat of 2 different videos.

is it a matter of input video specifications ? 
since its processes the video to dissolve shouldn't require 2 inputs so the first dissolves to the second ? 

any clarifications on usage would be really appreciated.

regards

**Comment by Akascape** [comment]
@yakuv0 Actually it depends on the settings and video. Some settings will give you no results. However I used the basic settings only. [BILAT, UMH, AOBMC]

Maybe it's because you have used the low quality gif I uploaded as the example. It only works with short videos having good amount of motion vectors. (to my experience)

Download the original mp4 video from here (click on the three dots):

https://user-images.githubusercontent.com/89206401/181591231-39f41888-bb6e-4d7d-9b8a-f8df722e6b58.mp4

Actually this was all inspired from [antonio](https://www.hellocatfood.com/motion-interpolation-for-glitch-aesthetics-using-ffmpeg-part-0/)'s work. I just created the GUI for it.
He actually slowed down the video speed by 50 which is super heavy to render.
